core/domain/wipeout_service.py

```diff
@@ -1241,26 +1345,22 @@ def _pseudonymize_models_transactional(
             if general_suggestion_model.final_reviewer_id == user_id:
                 general_suggestion_model.final_reviewer_id = pseudonymized_id
             general_suggestion_model.update_timestamps()
-
-        datastore_services.put_multi(
-            feedback_thread_models +
-            feedback_message_models +
-            general_suggestion_models)
+        datastore_services.put_multi(general_suggestion_models)
```

Comment:
Same note about not splitting.

Reply:
As mentioned above in the comment https://github.com/oppia/oppia/pull/15827#discussion_r945132048 I have added an Ignore here